plugins/modules/pkg5.py

```diff
@@ -54,6 +54,12 @@
       - Refresh publishers before execution.
     type: bool
     default: true
+  verbose:
+    description:
+      - Set to true to disable quiet execution.
```

Comment:
```suggestion
      - Set to V(true) to disable quiet execution.
```
(https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation)